tests/data/FeedbackFactory.ts

```diff
@@ -19,6 +19,10 @@ export class FeedbackFactory {
     };
   }
 
+  public static createEventFeedback(n: number): string[] {
+    return new Array(n).fill(faker.random.word());
```

Comment:
Should this be a random word or some value from a predefined array of values? IIRC feedback's supposed to be some predefined emojis.

Reply:
I'm not sure the predefined emojis have been confirmed yet since a lot of the design materials are moving around right now, and membership hasn't acknowledged them either since the portal redesign has been pushed back significantly (till after the store frontend is finished). I think it's fine to keep them at random words for now and then adjust later once the set of emojis /text is confirmed.